scripts/pinned_browsers.py

```diff
@@ -26,12 +26,12 @@ def calculate_hash(url):
     return h.hexdigest()
 
 
-def get_chrome_milestone():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--chrome_channel', default='Stable', help='Set the Chrome channel')
-    args = parser.parse_args()
-    channel = args.chrome_channel
-
+def get_chrome_milestone(channel=None):
+    if channel is None:
```

Comment:
Somewhere in the refactoring of this method something was removed. None of this code is needed, just set the default to 'Stable' and delete lines 30-34 here.